coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -1650,34 +1650,84 @@ def pad(context, node):
     context.add(res)
 
 
+@register_torch_op
+def adaptive_avg_pool1d(context, node):
+    _adaptive_pool1d(context, node, mb.reduce_mean)
+
+
 @register_torch_op
 def adaptive_avg_pool2d(context, node):
     _adaptive_pool2d(context, node, mb.avg_pool, mb.reduce_mean)
 
 
+def _adaptive_pool1d(context, node, reduce_op):
+    inputs = _get_inputs(context, node, expected=2)
+    x = inputs[0]
+    assert len(inputs[1].val) == 1
```

Comment:
nit:
same here.
Pron would be good to mention that the `output_size `argument should contain only one element